### ISSUE TITLE
Week Three Initial Assignment

### ISSUE THREAD

**Issue by vtpanda** [vtpanda/TiberDataScienceLearning#8]
3- Now that you have an optimized decision tree, try the following models: XGBoost (ooh! aah!), SVM, Naive Bayes, K-Nearest Neighbors, and Random Forest. (You'll need to pip install XGBoost, it doesn't come with SKLearn)

Plot all of their ROC curves against your optimized decision tree. Which one is best? Check out AUC and Log Loss. Does the dominant model dominate at all points on the ROC curve?

Did any of the models require additional or new/different preprocessing steps?
